bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/single_VLQ/NWA/TprimeBToTZ_LH/TprimeBToTZ_LH_narrowLO_M700/TprimeBToTZ_LH_narrowLO_M700_customizecards.dat

```diff
@@ -0,0 +1,8 @@
+set param_card mass 7000001 700
+set param_card bsm 1 0.100000e+00
+set param_card bsm 2 0.000000e+00
+set param_card bsm 3 0.100000e+00
+set param_card bsm 4 0.000000e+00
+set param_card bsm 5 0.100000e+00
+set param_card bsm 6 0.000000e+00
+set param_card DECAY 7000001 1.000000e-03
```

Comment:
how is the width defined

Reply:
In this case, we are considering the NWA scenario (width 1%), as I understood, it is not necessary to explicitly add the width. It is set to the default value of 1%, as in the model used for run2: : https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeToTZ/tZb_NW/tZ_b_LH_M700/tZ_b_LH_M700_customizecards.dat

Reply:
The width is an arbitrarily small number compared to the mass for the NWA. Anyway, there is an error in the particleId, now I have fixed it. Should I do another merge? Or do you prefer another way?